Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -89,6 +89,19 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
+    autobump_file_path = cask.tap.path/Tap::HOMEBREW_TAP_AUTOBUMP_FILE
+    if cask.tap.core_cask_tap? &&
+       autobump_file_path.exist? &&
+       File.readlines(autobump_file_path, chomp: true).include?(cask.name.first.downcase)
```

Comment:
> One odd thing (and the reason why used a regex in the first place): `cask.name` returns an array of uppercase cask names.

`cask.token` should be used instead I think:

```suggestion
       autobump_file_path.readlines(chomp: true).include?(cask.token)
```

```
brew(main):001:0> cask = Cask::CaskLoader.load("visual-studio-code")
=> #<Cask visual-studio-code /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/v/visual-studio-code.rb>
brew(main):002:0> cask.name
=> ["Microsoft Visual Studio Code", "VS Code"]
brew(main):003:0> cask.token
=> "visual-studio-code"
```

Reply:
That's it exactly.